recipes/optiwindnet/recipe.yaml

```diff
@@ -0,0 +1,100 @@
+context:
+  version: 0.1.1
+  python_min: 3.11
+  # on upgrade, update the sha256 hash
```

Comment:
You don't need this comment. Also, the bot will update it for you 99% of the time.
```suggestion
```